pkgs/by-name/et/etterna/package.nix

```diff
@@ -0,0 +1,113 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
+  copyDesktopItems,
+  # deps
+  clang,
+  cmake,
+  python2,
+  pkg-config,
+  openssl,
+  libGLU,
+  xorg,
+  alsa-lib,
+  libjack2,
+  libpulseaudio,
+  libogg,
+
+  # cfgs
+  enableCrashpad ? false,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "etterna";
+  version = "0.74.3";
+
+  src = fetchFromGitHub {
+    owner = "etternagame";
+    repo = "etterna";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-zzCk6axISswfTAk7rRha5HFzIHQ0AjpAZyAWzH+Cn1s=";
+  };
+
+  patches = [ ./fix-download-manager.patch ];
+
+  nativeBuildInputs = [
+    clang
```

Comment:
We usually try to keep builds working with both GCC and Clang, does this package depend on clang specifically?

Reply:
I don't think this specifically requires clang, moreso then my system uses clang and I put that there and forgot to remove it. Thanks for the reviews.

Reply:
By the official [`build instructions`](https://github.com/etternagame/etterna/blob/master/Docs/Building.md#clang) `clang` was needed because crashpad requires clang, and since we're removing that (since it also requires python2) this dependency should be lifted.